binder/mapping_test.go

```diff
@@ -29,3 +30,50 @@ func Test_EqualFieldType(t *testing.T) {
 	require.True(t, equalFieldType(&user, reflect.Int, "AGE"))
 	require.True(t, equalFieldType(&user, reflect.Int, "age"))
 }
+
+func Test_ParseParamSquareBrackets(t *testing.T) {
+	tests := []struct {
```

Comment:
@dojutsu-user  With the Makefile in the root of the repo run `make betteralign`, then `make lint`.

Reply:
Thanks for the tip @gaby. I have updated the changes.